source/includes/fact-export-favorite-connection.rst

```diff
@@ -0,0 +1,6 @@
+When you export favorite connections, |compass-short| exports the list
+of connections as a JSON file.
+
+By default, when you export favorite connections, database usernames and
+passwords are included in plaintext. To protect sensitive connection
+information, you may encrypt the exported file with a passphrase.
```

Comment:
Consider removing "you may", to be more direct.

Reply:
Updated.